core-navigation/src/main/java/com/rocket/android/core/navigation/extension/FragmentExt.kt

```diff
@@ -113,13 +130,21 @@ internal fun Fragment.safeState(ignore: Boolean = false, action: (fragment: Frag
 }
 
 @PublishedApi
-internal fun Fragment.popBack() {
+internal fun Fragment.popBack(@IdRes id: Int? = null, inclusive: Boolean? = null) {
     var fragment: Fragment? = this
-    while (fragment != null && !fragment.findNavController().popBackStack()) {
+    while (fragment != null && !fragment.navigateBack(id, inclusive)) {
         fragment = fragment.parentFragment
     }
     if (fragment == null) {
         activity?.finish()
     }
 }
+
+internal fun Fragment.navigateBack(@IdRes id: Int? = null, inclusive: Boolean?): Boolean {
+    return id?.let {
+        this.findNavController().popBackStack(id, inclusive ?: false)
```

Comment:
Why use ?:false instead use inclusive: Boolean? = false in constructor?, in another methods we do that. Just curious